src/cpp/database/database_queue.hpp

```diff
@@ -575,9 +572,12 @@ class DatabaseDataQueue : public DatabaseQueue<std::shared_ptr<T>>
     std::string deserialize_guid(
             StatisticsGuid data) const
     {
-        eprosima::fastrtps::rtps::GUID_t guid;
-        memcpy(guid.guidPrefix.value, data.guidPrefix().value().data(), eprosima::fastrtps::rtps::GuidPrefix_t::size);
-        memcpy(guid.entityId.value, data.entityId().value().data(), eprosima::fastrtps::rtps::EntityId_t::size);
+        eprosima::fastdds::
+                rtps::GUID_t guid;
```

Comment:
Check for more ocurrances like this one and fix them
```suggestion
        eprosima::fastdds::rtps::GUID_t guid;
```